data/templates/ids/fastnetmon_excluded_networks_list.j2

```diff
@@ -0,0 +1,5 @@
+{% if excluded_network is vyos_defined() %}
```

Comment:
No need for `()`, `vyos_defined` is sufficient?

Reply:
I was looking at the fastnetmon_networks_list.j2 which does it the same way. If its not required i can updated the pull request without it.

Reply:
I have made the change, but dindt fixed it on fastnetmon_networks_list.j2 because it's not related to this pull request